README.md

```diff
@@ -228,28 +228,40 @@ destruction, or when newer block replacing old one.
 
     Usage: rdb-cli /path/to/dump.rdb [OPTIONS] {json|resp|redis} [FORMAT_OPTIONS]
     OPTIONS:
-    -k, --filter-key <REGEX>      Filter keys using regular expressions
-    -l, --log-file <PATH>         Path to the log file (Default: './rdb-cli.log')
+            -l, --log-file <PATH>         Path to the log file (Default: './rdb-cli.log')
     
+            Multiple filters combination of keys/types/dbs can be specified:
+            -k, --key <REGEX>             Filter keys using regex
```

Comment:
Include only this keys that match REGEX. Maybe I should modify the description to be:
```
-k, --key <REGEX>             Include only keys that match REGEX.
```

Filters and handlers are being piled and called in reverse order to their registration. The CLI
takes care to register resp/redis handlers and only then the filters. Among the filters
it is not importent their order because we shall get anyway the same result.

Reply:
so as soon as someone uses `--key` it'll change the default and implicitly skip all other keys?
and when someone adds `--no-key` it'll keep the default and skip just that key?
what happens if someone mixes these options? or are there mutually exclusive.
either way, make sure the docs are clear.

Reply:
The misunderstanding might start with the word `change`. We change nothing just adding yet another filter to the pipeline.
Note that there is also one line description in the Usage `Multiple filters combination of keys/types/dbs can be specified:`. I will try to clarify this point. thanks.